src/Mod/CleanStart/Gui/qml.qrc

```diff
@@ -0,0 +1,5 @@
+<RCC>
+    <qresource prefix="/">
+        <file>CleanStart.qml</file>
```

Comment:
@chennes 
Shouldn't this file be part of the PR?

Reply:
Thanks for the catch -- I thought I had removed all of the QML, but I forgot about the resources. I'll clean that up.